Project1/Task10-homework/Collections1.cs

```diff
@@ -0,0 +1,243 @@
+            }
+            GetMaxValue(queue);
+
+            //Write a program that takes three letters as input and displays them in reverse order. Use Stack.
+            Console.WriteLine("---stack---");
+            var myStack = new Stack<int>();
+            Console.WriteLine("Enter values:");
+            Console.WriteLine("First value:");
+            myStack.Push(int.Parse(Console.ReadLine()));
+            Console.WriteLine("Enter the second value:");
+            myStack.Push(int.Parse(Console.ReadLine()));
+            Console.WriteLine("Enter the second value:");
+            myStack.Push(int.Parse(Console.ReadLine()));
+            Console.WriteLine("in reverse order:");
+            foreach (var item in myStack)
+            {
+                Console.WriteLine(item);
+            }
+
+            //Dictionary
+            //Add a new value whose key is your name and whose value is your age. Do this using the Add method.Then addanother value to the dictionary using index notation. This time, use a different name and a different age.Finally, read the first item you added to the dictionary and write it to the console using Console.WriteLine.
+
+            var myDic = new Dictionary<string, int>();
+            myDic.Add("Tolganay", 22);
+            myDic["Alila"] = 22;
+            Console.WriteLine($"key : {myDic.ElementAt(0).Key} ||  value: {myDic.ElementAt(0).Value}");
+
+            //Create two lists, each with 10 values.The first list is of type int, where the values are not in order.the second list is of type string, the values are also not alphabetically specified. Write a method that performs sorting operations on the two lists the int list in ascending and the string list in descending order. Then this method merges the lists into a dictionary.Output the resulting word to the console *
+            var myListF = new List<int>() { 4, 3, 7, 8, 6 };
+            var myListG = new List<string>() { "aka", "maka", "saka", "lola", "kola" };
+            SortMerge(myListF, myListG);
+
+            //            Create a City class where there are fields int population, double area.Create a dictionary where Key is the name of the city and Value is the corresponding name of the cityand the object of type City. Create 5 elements for the dictionary. 
+            //Sort the dictionary by city area and display it on your console
+            //Browse the dictionary by population in reverse order and display it on your console
+            //Count the total population of all cities and output to the console *
+            Dictionary<string, City> myCity = new Dictionary<string, City>();
+            myCity.Add("Oku", new City(23, 456.5));
+            myCity.Add("Aka", new City(34, 345.4));
+            myCity.Add("mika", new City(24, 355.2));
+            myCity.Add("Lala", new City(44, 445.4));
+            myCity.Add("Saka", new City(54, 555.2));
+            var sorted = myCity.OrderBy(x => x.Value.Area);
```

Comment:
this is alredy not a dictionary, it's `System.Collections.Generic.KeyValuePair<int,int>`